Plugin.NetStandardStorage/Implementations/Folder.cs

```diff
@@ -236,8 +236,9 @@ public IList<IFolder> GetFolders()
                 .AsReadOnly();
         }
 
-        public bool CheckFileExists(string path)
+        public bool CheckFileExists(string fileName)
         {
+            var path = Path.Combine(FullPath, fileName);
             return System.IO.File.Exists(path);
```

Comment:
new line here, before the return. I want initialization to be separated vertically from other concepts, like return in this case